OmsAgent/omsagent.py

```diff
@@ -328,6 +370,12 @@ def install():
     """
     exit_if_vm_not_supported('Install')
 
+    try:
+        if runShellBundleSigningVerification:
```

Comment:
should this be in main() instead of install()

Reply:
done